src/bpftrace.cpp

```diff
@@ -324,6 +324,11 @@ void BPFtrace::request_finalize()
 {
   finalize_ = true;
   attached_probes_.clear();
+
+  for (int pid : child_pids_)
+  {
+    kill(pid, SIGTERM);
```

Comment:
If we really wanna kill the child, is SIGTERM sufficient? Not sure if it's a good idea but would exiting and orphaning the child work?

Reply:
I thought about using SIGKILL, but it seems overkill. I don't think orphaning the child is a good idea though.

Reply:
Sure, sounds good. I guess if peoples SIGTERM handler is broken it's not like they can expect a clean bpftrace exit either.